content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -64,13 +77,28 @@ admissionConfig:
         apiVersion: apiserver.config.k8s.io/v1alpha1
         kubeConfigFile: /dev/null
         kind: WebhookAdmission
+
 {{< /text >}}
 
 In the same directory, execute:
 
 {{< text bash >}}
+
```

Comment:
Remove leading and trailing blank lines inside `text bash` sections.

Reply:
Thanks @frankbu. Getting some weird linter issues with this PR and that's where the blank lines are coming from. If I remove the blank lines, the linter complains that I need to have a blank line before and after the tag and mentions a whole bunch of spelling errors for the oc commands. If I add the blank lines in, I get different errors.

Reply:
Problem was {{/text}} should have been {{< /text >}}